filament/backend/src/opengl/PlatformWGL.cpp

```diff
@@ -249,6 +249,10 @@ void PlatformWGL::makeCurrent(Platform::SwapChain* drawSwapChain,
     }
 }
 
+void PlatformWGL::makeCurrent() noexcept {
+    assert_invariant(false);
```

Comment:
Can't we call `wglMakeCurrent(0, mContext);`? Although not that important for our application.

Reply:
"The hdc parameter must refer to a drawing surface supported by OpenGL" (Source: https://docs.microsoft.com/en-us/windows/win32/api/wingdi/nf-wingdi-wglmakecurrent)